Libraries/Components/TextInput/TextInput.js

```diff
@@ -1098,6 +1098,7 @@ const TextInput = createReactClass({
     if (this.props.onFocus) {
       this.props.onFocus(event);
     }
+    this.focus();
```

Comment:
Can you flip the order and put this above the if statement to be consistent with `_onBlur`?

https://github.com/facebook/react-native/blob/6ba2769f0f92ca75fb0eb60ccb8337920a9c31eb/Libraries/Components/TextInput/TextInput.js#L1182-L1189

I think it's important because if someone checks if they are the currently focused input in the `onFocus` prop they pass in it should be true.

Reply:
Actually, I'm able to just edit the branch on your remote through the GitHub UI. Neat!